### DISCUSSION TITLE
Where to store Histoire stories....

### DISCUSSION THREAD

**Discussion by erondpowell** [histoire-dev/histoire#365] [Q&A]
I noticed in the stackblitz guides, `.story.vue` files are alongside the components. I'm worried this pattern could get visually cluttered over time and I'm curious if its a good idea to create them in a separate `histoire` directory. 

I'm also wondering if there's any previously pioneered way to automate story-building. For example, I'm thinking to use `fs`, iterate over each sfc, then create new stories automagically... But I'm a "pseudo-dev", still in learning mode... so it'd be great to see someone else's work.

**Comment by paulbgomez** [comment]
I made a separate histoire folder with its own config and the stories inside it. 
You can also wrap your components in isolated folders, and have inside them the component.vue and the story.vue; this way you can share the project config files.
